src/components/styledComponents.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import { styled } from "nativewind";
+import {
+  Image as StyledImage,
+  KeyboardAvoidingView as StyledKeyboardAvoidingView,
+  Pressable as StyledPressable,
+  SafeAreaView as StyledSafeAreaView,
+  ScrollView as StyledScrollView,
+  Text as StyledText,
+  TextInput as StyledTextInput,
+  View as StyledView
```

Comment:
Not a blocker, but I would do this as `View as OriginalView`. The return value of the `styled()` function is presumably "styled", not the input.